kolibri/plugins/management/assets/src/device_management/views/index.vue

```diff
@@ -2,6 +2,11 @@
 
   <core-base :topLevelPageName="DEVICE" :appBarTitle="$tr('deviceManagementTitle')">
     <div>
+
+      <transition name="delay-entry">
```

Comment:
Transition adds a fade in animation at .75s (a bit longer than the default), per @jtamiace's redline spec